read.go

```diff
@@ -215,7 +215,7 @@ func readSlice(r *bufio.Reader, sz int, v reflect.Value) (int, error) {
 	return sz, nil
 }
 
-func readFetchResponseHeader(r *bufio.Reader, size int) (throttle int32, remain int, err error) {
+func readFetchResponseHeader(r *bufio.Reader, size int) (watermark int64, throttle int32, remain int, err error) {
```

Comment:
Should the watermark come after the throttle value since it's given after in the response? Seems more natural to preserve the order.